doc/source/user_guide/index.rst

```diff
@@ -4,29 +4,39 @@
 User guide
 ==========
 
+Use the Rocky PrePost Scripting
+-------------------------------
```

Comment:
```suggestion
Use Rocky PrePost Scripting
---------------------------
```
Same question about whether "Scripting" should be capitalized. Only capitalize it if it is a proper noun.